.changeset/dry-cups-train.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-css": major
+---
+
+All styles have been wrapped in a cascade layer named `itwinui`.
```

Comment:
Should we mention the v3 nested cascade layer (`itwinui.v3`)?

Reply:
Probably not? It's an implementation detail that we should be able to change at any time.